### DISCUSSION TITLE
AIR 50.2.3.5 ANT build simple Flex Desktop app - preparation for GitHub / Actions

### DISCUSSION THREAD

**Discussion by Jasenvm** [airsdk/Adobe-Runtime-Support#3044] [Q&A]
I have started the adventure to build my relatively simple Flex Desktop app using using github Actions for the first time.

I hope this is the correct place to post such an issue.

So, I started looking into using ant to build my app using https://airsdk.dev/docs/tools/building/ant/examples and other tutorials as a guide.

So far having a couple of weird issues with the `build.xml` script:

<img width="913" alt="image" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/66523801/26071128-c576-4329-bc77-c67f78fe9bbe">

This simple script fails to resolve `source-path` property. I have ANT_HOME, FLEX_HOME and AIR_TOOLS setup correctly.

Also, I am using IntelliJ 2023.3.2 and AIR 50.2.3.5 SDK merged with Flex_4.16, all seems fine.

Also, notice I had directly referenced the `flextask.jar` file to get the `taskdefs` to resolve, so I assume I have some kind of core setup issue not yet correct. Just weird that it partially 'works'.

Any ideas what I am doing wrong much appreciate.

**Comment by 2jfw** [comment]
Did you actually run the script and see if it works anyways?
We are also building our apps on Jenkins with `Ant` and `build.xml`s and in IntelliJ I'm seeing a couple of unresolvable tags and variables (including `<source-path`). I would think that you can just ignore these as all our apps build successfully on CI though IntelliJ warnings/errors
